include/circt/Dialect/SV/SV.td

```diff
@@ -37,6 +37,11 @@ def NonProceduralOp : NativeOpTrait<"NonProceduralOp"> {
   let cppNamespace = "::circt::sv";
 }
 
+/// Mark an operation as being a vendor extension.
+def VendorExtension : NativeOpTrait<"VendorExtension"> {
+  let cppNamespace = "::circt::sv";
+}
```

Comment:
Great idea to put this in an op trait!

It might work very well to use traits to factor what is supported by different Verilog standards as well. 🤔

Reply:
Yea, I was thinking in that direction.  There was the version of this I never got to work right which took a string for which vendor's extension it was, but that is probably too little detail for when we really care about this.